proposals/CSP.md

```diff
@@ -109,8 +91,9 @@ The function returns a `WebAssembly.Instance` that allows executing the
 WebAssembly code. If WebAssembly bytes are provided, `instantiate` will first
 perform the steps of `WebAssembly.compile`.
 
-_Risks:_ loads executable code into the running program. This code is confined,
-being only able to access objects reachable from the import object. The instance
+_Risks:_ loads executable code into the running program and execute any included `start` function.
```

Comment:
Again some DoS risks from compilation because compilation may in principle be deferred until instantiation.

Reply:
I think a principled approach here would be to say that we should gate a privileged activity at the earliest place it can happen, even if it might happen later in a given application. That said, whether DoS attacks are in scope remains as a question.